### DISCUSSION TITLE
Replace / add In addition to the slider, an input window in full watts

### DISCUSSION THREAD

**Discussion by Thunder1805** [thomluther/ha-anker-solix#158] [Ideas]
Would it be possible to have add to the slider, adjusting the Output value of s Solar power from Batteries, a BOX to put in full values? While handle the slider with fingers on Handy it is critical to have the exact planted Value to adjust. 
![slider1](https://github.com/user-attachments/assets/3eaa1c6c-c5af-43e1-b958-cf0c825b1cde)

**Comment by thomluther** [comment]
Hi @Thunder1805  I believe this was answered already somewhere else. It is up to you how you present a number entity in your dashboard. There are many different cards available.
The default representation of the entity in the integration device is a slider on purpose, since default input fields have also increment buttons which I do NOT recommend to use, because each increment is a change via the API. Increases are limited to 30 sec intervals to limit Api traffic.
